src/Analyzers/Core/Analyzers/SimplifyInterpolation/Helpers.cs

```diff
@@ -90,9 +90,14 @@ private static IOperation Unwrap(IOperation expression)
                     return;
                 }
 
-                if (invocation.Arguments.Length == 0 &&
-                    (invocation.TargetMethod.ContainingType.SpecialType == SpecialType.System_Object ||
-                    invocation.TargetMethod.OverriddenMethod?.ContainingType.SpecialType == SpecialType.System_Object))
+                var method = invocation.TargetMethod;
```

Comment:
in general i woudl have asked to doc why this is needed.  but it's no biggie here :)

Thanks for the contribution!